sprout/views/column_review.py

```diff
@@ -0,0 +1,50 @@
+"""File with column_review view."""
+
+from django.shortcuts import get_object_or_404, redirect, render
+from django.urls import reverse
+
+from sprout.forms import ColumnMetadataForm
+from sprout.models import ColumnMetadata, TableMetadata
+
+
+def column_review(request, table_id):
+    """Takes the data from ColumnMetadata and displays as a table.
+
+    The table can be edited and the result written back to the db
+
+    Args: Must learn what to write here
+        request (_type_): _description_
+        table_name (_type_): _description_
+
```

Comment:
```suggestion
        request: _description_
        table_name: _description_
        
```
Remove ` (_type_)` from docstrings in general :)